vibration_toolbox/continuous_systems.py

```diff
@@ -90,6 +91,7 @@ def euler_beam_modes(n=10, bctype=2, npoints=2001,
     # len=[0:(1/(npoints-1)):1]';  %Normalized length of the beam
     len = np.linspace(0, 1, npoints)
     x = len * L
+    print(x)
```

Comment:
This print statement can make tests fail since this is not a value expected at the docstring.